keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone.py

```diff
@@ -0,0 +1,518 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""MobileNet v3 backbone model for KerasCV.
+
+References:
+    - [Searching for MobileNetV3](https://arxiv.org/pdf/1905.02244.pdf)
+    (ICCV 2019)
+    - [Based on the original keras.applications MobileNetv3](https://github.com/keras-team/keras/blob/master/keras/applications/mobilenet_v3.py)
+"""  # noqa: E501
+
+import copy
+
+from tensorflow import keras
+from tensorflow.keras import backend
+from tensorflow.keras import layers
+from tensorflow.keras.utils import custom_object_scope
+
+from keras_cv import layers as cv_layers
+from keras_cv.models import utils
+from keras_cv.models.backbones.backbone import Backbone
+from keras_cv.models.backbones.mobilenet_v3.mobilenet_v3_backbone_presets import (  # noqa: E501
+    backbone_presets,
+)
+from keras_cv.utils.python_utils import classproperty
+
+channel_axis = -1
+
+
+def adjust_channels(x, divisor=8, min_value=None):
+    """Ensure that all layers have a channel number that is divisible by the
+    `divisor`.
+
+    Args:
+        x: integer, input value.
+        divisor: integer, the value by which a channel number should be
+            divisible, defaults to 8.
+        min_value: float, minimum value for the new tensor.
```

Comment:
float, optional. Minimal value for the new tensor. If `None`....

Reply:
Changed